cluster/gce/gci/README.md

```diff
@@ -1,11 +1,42 @@
-# Container-VM Image
+# Container-Optimized OS
 
-[Container-VM Image](https://cloud.google.com/compute/docs/containers/vm-image/)
-is a container-optimized OS image for the Google Cloud Platform (GCP). It is
-primarily for running Google services on GCP. Unlike the open preview version
-of container-vm, the new Container-VM Image is based on the open source
-ChromiumOS project, allowing us greater control over the build management,
+[Container-Optimized OS](https://cloud.google.com/container-optimized-os/docs),
+(previously Google Container-VM image a.k.a GCI) is a container-optimized OS image for the Google Cloud Platform (GCP). It is
+primarily for running Google services on GCP. Container-Optimized OS is an open
+source OS based on
+the open source [ChromiumOS project](https://www.chromium.org/chromium-os), allowing us greater control over the build management,
 security compliance, and customizations for GCP.
 
+Container-Optimized OS is open [source](https://cos.googlesource.com), and is released on milestones. Example milestones are
+81,  84, 85. Each milestone will experience three release channels -- dev, beta and stable to reach
```

Comment:
Not sure what to take from the "Example milestones are ######" sentence.

Reply:
This shows what number 81,  85 mean.  Any good suggestion?